backend/api/Areas/Reports/Models/Project/ProjectModel.cs

```diff
@@ -27,12 +27,12 @@ public class ProjectModel
         /// <summary>
         /// get/set - The reported fiscal year this project.
         /// </summary>
-        public int ReportedFiscalYear { get; set; }
+        public string ReportedFiscalYear { get; set; }
```

Comment:
Any reason for the string requirement?  Isn't it just an `int`?

Reply:
The output for the reported fiscal year is actually a string: "21/22"

Reply:
Ok, it looks like my change has affected some of the backend tests....